.travis/lib-setup.sh

```diff
@@ -78,7 +78,7 @@ setup::macos::install() {
 
 setup::windows::install() {
     # Install the right Python version and MIT Kerberos
-    choco install python"${PYENV:0:1}" --version $PYENV
+    choco install python"${PYENV:0:1}" --version $PYENV || true
```

Comment:
From the log linked in the commit log:

```
Progress: Downloading python3 3.8.1... 100%

python3 v3.8.1 [Approved]
python3 package files install completed. Performing other installation steps.
Installing 64-bit python3...
WARNING: Generic MSI Error. This is a local environment error, not an issue with a package or the MSI itself - it could mean a pending reboot is necessary prior to install or something else (like the same version is already installed). Please see MSI log if available. If not, try again adding '--install-arguments="'/l*v c:\python3_msi_install.log'"'. Then search the MSI Log for "Return Value 3" and look above that for the error.
ERROR: Running ["C:\ProgramData\chocolatey\lib\python3\tools\python-3.8.1-amd64.exe" /quiet InstallAllUsers=1 PrependPath=1 TargetDir="C:\Python38" ] was not successful. Exit code was '1603'. Exit code indicates the following: Generic MSI Error. This is a local environment error, not an issue with a package or the MSI itself - it could mean a pending reboot is necessary prior to install or something else (like the same version is already installed). Please see MSI log if available. If not, try again adding '--install-arguments="'/l*v c:\python3_msi_install.log'"'. Then search the MSI Log for "Return Value 3" and look above that for the error..
The install of python3 was NOT successful.
Error while running 'C:\ProgramData\chocolatey\lib\python3\tools\chocolateyInstall.ps1'.
 See log for details.

Chocolatey installed 0/1 packages. 1 packages failed.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
```

More that I could find on this particular error code:
https://support.microsoft.com/en-us/help/834484/you-receive-an-error-1603-a-fatal-error-occurred-during-installation

I have seen something similar when testing on Windows (AppVeyor, GitHub CI) but installing msi files directly, not via Chocolatey. The ones I've seen have always been an installer which wants to reboot the system. IIRC Python's installer wants to reboot Windows so that changes to %PATH% are taken into account (or something along those lines). 

The exit code from choco here is not 0 so I'm ignoring it, otherwise we can't proceed.